Makefile

```diff
@@ -110,6 +110,16 @@ test-gitopsservice-nondefault:
 test: manifests generate fmt vet ## Run unit tests.
 	go test `go list ./... | grep -v test` -coverprofile cover.out
 
+.PHONY: e2e-tests-sequential
+e2e-tests-sequential:
```

Comment:
Please add a line comment after the target name so the `make help` can show the comment/help. Take a look at the `uninstall` target for example

Reply:
Done changes! Thanks